src/subdomains/supporting/payment/services/transaction-helper.ts

```diff
@@ -250,6 +252,20 @@ export class TransactionHelper implements OnModuleInit {
       maxVolumeTarget: txSpecTarget.maxVolume ?? undefined,
       minVolumeTarget: txSpecTarget.minVolume,
       fee,
+      feeSource: {
+        ...fee,
+        blockchain: txSpecSource.blockchainFee,
+        fixed: txSpecSource.fixedFee,
+        min: txSpecSource.minFee,
+        total: txSpecSource.totalFee,
+      },
```

Comment:
This will assign all props of `fee` to the result and return it (for an API call). There is the payoutRefBonus in the fee?